.github/workflows/nightly.yml

```diff
@@ -2,64 +2,49 @@ name: Nightly
 on:
   schedule:
    - cron: "0 1 * * *"
```

Comment:
```suggestion
```
The inputs would be missing here.

Reply:
Oops, good catch. Forgot to remove this. :slightly_smiling_face: